src/store/reducer.js

```diff
@@ -0,0 +1,23 @@
+const initialState = {
+  isLoggedIn: false,
+  user: null,
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'LOGIN_STATE':
+      return {
+        ...state,
+        isLoggedIn: Boolean,
+      };
```

Comment:
please make sure that the linting rules require you to have a space between the cases, to make the code more readable. you can check the AirBnb style guide for this, and make sure that you have extended linting rules from AirBnb.